cms/static/cms/js/modules/cms.modal.js

```diff
@@ -666,6 +672,32 @@
 
                 // update title
                 this.ui.titlePrefix.text(el.text());
+            },
+
+            /**
+             * _deletePlugin removes a plugin once created when clicking
+             * on delete or the close item. If we don't do this, an empty
+             * placeholder is generated
+             * https://github.com/divio/django-cms/pull/4381 will eventually
+             * provide a better solution
+             *
+             * @param hideAfter Boolean
```

Comment:
not to be picky, but can we avoid the so called "[boolean trap](http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html)"? 
would be so much easier with `_deletePlugin({ hideAfter: true })`